### ISSUE TITLE
How to use google redis / memorystore service?

### ISSUE THREAD

**Issue by basiszwo** [GoogleCloudPlatform/gcp-service-broker#467]
Google Redis / MemoryStore is listed to be a preview service. As we have enabled to see preview services in our services list (pcf marketplace) we were expecting to see google redis as an available service. Unfortunately it's not listed. 

We saw that `GSB_COMPATIBILITY_ENABLE_TERRAFORM_SERVICES` exists and updated it to `true`.

As a result we did not see any change.  

The question is how we can use google redis? Can someone explain this to me?

Thank you!

**Comment by josephlewis42** [comment]
Hi @basiszwo,

I apologize for this being confusing. In order to enable Redis right now you have to give the service broker a "brokerpak" which is a way to side-load on services built using Terraform.

The long term vision is to release a service-broker without any cloud-specific logic and can just work with Terraform modules. I gave a (terrible) [talk at CF Summit](https://www.youtube.com/watch?v=BXIvzEfHil0&list=PLhuMOCWn4P9h-9tcBVRFCaQ7rmdof66pe&index=44&t=1s) with more details.

If you can wait about a week, @sophiawho is working on making Redis a built-in service because we know it's an important one to have core until the long-term vision pans out. If you need it sooner I can build you a brokerpak with Redis for trying it out and walk you through using it.

Thanks
\- Joseph

**Comment by marcohelmerich** [comment]
Hi @josephlewis42,
we managed to build the brokerpak and loaded it in the broker. @basiszwo we will update you monday on this. If we find the time on monday we would also add a PR with some documentation on how to actually use the brokerpaks. The idea itself is great and is very promising. 

We also found I would not say  a bug but some kind of inconvenience with the default network  for redis and added a proposal to make this more convenient here:
https://github.com/GoogleCloudPlatform/gcp-service-broker/pull/470
